packages/cli/templates/Update.spec.ts

```diff
@@ -0,0 +1,348 @@
+			};
+			const upgradable: PackageDefinition = {
+				trial: reactPkgResolve.NPM_REACT,
+				licensed: reactPkgResolve.FEED_REACT
+			};
+			spyOn(reactPkgResolve, "getReactUpgradeablePackages").and.returnValue([upgradable]);
+			(fsSpy.directoryExists as jasmine.Spy).and.returnValue(false);
+			(fsSpy.readFile as jasmine.Spy).and.returnValue(JSON.stringify(mockJSONInput, null, 4));
+			spyOn(PackageManager, "ensureRegistryUser").and.returnValue(false);
+			expect(await updateWorkspace("")).toEqual(false);
+		});
+
+		it("Should update package.json file, removing non-scoped igniteui-react packages", async () => {
+			const mockPackageJSON = {
+				dependencies: {
+					"@alphabetically-sorted-scope/package": "^0.0.0",
+					"alphabetically-second-package": "^0.0.0",
+					"igniteui-react": "^18.5.1",
+					"igniteui-dockmanager": "^1.0.0",
+					"some-package": "^0.0.0"
+				}
+			};
+			(fsSpy.fileExists as jasmine.Spy).and.returnValue(true);
+			(fsSpy.glob as jasmine.Spy).and.returnValues([]);
+			(fsSpy.readFile as jasmine.Spy).and.callFake((filePath: string) => {
+				if (filePath.indexOf("package.json") > -1) {
+					return JSON.stringify(mockPackageJSON);
+				}
+			});
+			spyOn(PackageManager, "ensureRegistryUser").and.returnValue(true);
+			spyOn(Util, "log");
+			expect(await updateWorkspace("")).toEqual(true);
+			expect(fsSpy.writeFile).toHaveBeenCalledWith("package.json", Util.formatPackageJson({
+				dependencies: {
+					"@alphabetically-sorted-scope/package": "^0.0.0",
+					"@infragistics/igniteui-react": "^18.5.1",
+					"@infragistics/igniteui-dockmanager": "^1.0.0",
+					"alphabetically-second-package": "^0.0.0",
+					"some-package": "^0.0.0"
+				}
+			}));
+			expect(fsSpy.writeFile).toHaveBeenCalledTimes(2);
+			expect(fsSpy.glob).toHaveBeenCalledTimes(1);
+		});
+
+		it("Should update import paths in files correctly", async () => {
+			const mockFileArray: MockFile[] = [
+				{
+					path: "package.json",
+					content:
+					`{
+  "dependencies": {
+      "igniteui-dockmanager": "^1.0.0",
+      "igniteui-react": "^18.5.1",
+      "igniteui-react-grids": "^18.5.1",
+      "some-package": "^0.0.0"
+	}
+}
+	`,
+					expected:
+					`{
+  "dependencies": {
+    "@infragistics/igniteui-dockmanager": "^1.0.0",
+    "@infragistics/igniteui-react": "^18.5.1",
+    "@infragistics/igniteui-react-grids": "^18.5.1",
+    "some-package": "^0.0.0"
+  }
+}
+`
+				},
+	{
+				path: "src/home.tsx",
+				content:
+`import { something } from 'module';
+import { bait } from 'igniteui-react-core';
+import 'igniteui-react-grids/grids';
+import { IgrGridModule, IgrGrid, IgrColumn } from 'igniteui-react-grids';
+import { IgcDockManager } from 'igniteui-dockmanager';
+import 'igniteui-react-grids/grids/themes/light/bootstrap.css'
+
+IgrGridModule.register();
+
+export default function Home() {
+	const title = 'igniteui-react example';
+}`,
+				expected:
+`import { something } from 'module';
+import { bait } from 'igniteui-react-core';
```

Comment:
Shouldn't 'igniteui-react-core' be licensed?

Reply:
It is not in the MockFile dependencies and that's why it is not licensed. I'll add it to dependencies in order to be licensed after upgrade.